data/docs/monitor-http-endpoints.mdx

```diff
@@ -1,146 +1,287 @@
+    <figcaption><i>Configuring httpcheck.status query with WHERE clause</i></figcaption>
+</figure>
+
+5. You can also Average By specific endpoints using the `http.url` attribute
+
+### Step 3: Visualize Endpoint Health
+
+The resulting chart will show:
+- Value of 1 when endpoints are healthy (returning 2xx status codes)
+- Value of 0 when endpoints are down or returning error codes
+- Different lines for each monitored endpoint
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/monitor-http-endpoints/endpoint-health-chart.webp" alt="HTTP endpoint health visualization"/>
+    <figcaption><i>HTTP endpoint health visualization showing status over time</i></figcaption>
+</figure>
+
+## Available Metrics
+
+The HTTP Check receiver provides several metrics:
+
+| Metric | Description | Type |
+|--------|-------------|------|
+| `httpcheck.status` | HTTP response status (1 for matching status class, 0 otherwise) | Sum |
+| `httpcheck.duration` | Duration of the HTTP request in milliseconds | Gauge |
+| `httpcheck.error` | Number of errors during HTTP checks | Sum |
+
+## Available Attributes
+
+Key attributes for filtering and grouping:
+
+| Attribute | Description | Example Values |
+|-----------|-------------|----------------|
+| `http.url` | Full HTTP request URL | `http://example.com`, `http://my-app.com/health` |
+| `http.status_code` | HTTP response status code | `200`, `404`, `500` |
+| `http.method` | HTTP request method | `GET`, `POST`, `PUT` |
+| `http.status_class` | HTTP response status class | `2xx`, `4xx`, `5xx` |
+
+For more details about the metrics and their related attributes, refer to the [HTTP Check Receiver documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/httpcheckreceiver/documentation.md).
+
+## Setting Up Alerts
+
+You can create alerts to get notified when endpoints go down:
+
+### Step 1: Create Alert from Dashboard
+
+1. In your dashboard panel showing HTTP endpoint health, click the dropdown menu
+2. Select **Create Alerts**
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/monitor-http-endpoints/create-alert-dropdown.webp" alt="Creating alert from dashboard panel"/>
+    <figcaption><i>Creating alert from dashboard panel dropdown</i></figcaption>
+</figure>
+
+3. This will take you to the alerts configuration page
+
+### Step 2: Configure Alert Conditions
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/monitor-http-endpoints/alert-configuration.webp" alt="Alert configuration page"/>
+    <figcaption><i>Alert configuration page with conditions setup</i></figcaption>
+</figure>
+
+Example alert configuration:
+```yaml
+Metrics: httpcheck.status         
+WHERE: http.status_class = 2xx 
+Avg By: http.url
+
+
+condition: below the threshold at least once during the last 5 mins
+threshold: 1   
+```
+
+The alert fires when any monitored endpoint fails to return even a single 2xx (success) response during a 5 minute window.
```

Comment:
Wording is confusing.
Improve it to:

The alert fires when any monitored endpoint does not respond with 2xx (success) response (even once) during a 5 minute window.